lib/msal-common/src/authority/AzureRegionConfiguration.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AzureRegion } from "./AzureRegion";
+
+/*
+ * AzureRegionConfiguration
+ * - preferredAzureRegion       - Preferred azure region from the user 
+ * - environmentRegionFunc      - Environment specific way of fetching the region from the environment
+ */
+export type AzureRegionConfiguration = {
+    preferredAzureRegion?: AzureRegion,
+    environmentRegionFunc: () => string | undefined; 
```

Comment:
Why does this need to be a function? Can you just pass in the string?

Reply:
That's a good point, I agree we don't need to pass in a function we can just pass in the string.